app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/ConversionHostsSettings.js

```diff
@@ -55,17 +61,37 @@ class ConversionHostsSettings extends React.Component {
             {conversionHosts.length === 0 ? (
               <ConversionHostsEmptyState showConversionHostWizardAction={showConversionHostWizardAction} />
             ) : (
-              <ConversionHostsList conversionHosts={conversionHosts} />
+              <ConversionHostsList
+                conversionHosts={conversionHosts}
+                setHostToDeleteAction={setHostToDeleteAction}
+                showConversionHostDeleteModalAction={showConversionHostDeleteModalAction}
+                showConversionHostDeleteModal={showConversionHostDeleteModal}
+                conversionHostToDelete={conversionHostToDelete}
+                hideConversionHostDeleteModalAction={hideConversionHostDeleteModalAction}
+                yesToDeleteConversionHostAction={yesToDeleteConversionHostAction}
+              />
             )}
             {conversionHostWizardMounted && <ConversionHostWizard />}
           </React.Fragment>
         )}
       </Spinner>
     );
   }
+
+  componentDidUpdate = prevProps => {
```

Comment:
Like we mentioned on the standup today, I think we can leave this spinner out if the request is async, since progress for that will be reported in the list view instead of the modal.

Reply:
I moved the API call to `onClick` handler of `
DeleteConversionHostConfirmationModal` as you suggested.